packages/client/src/components/MainLayout/index.tsx

```diff
@@ -1,12 +1,49 @@
 import React, { Component, ReactNode } from 'react'
+import HeaderComponent from '../Header'
 import './index.css'
 
 type MainLayoutProps = {
   children: ReactNode
+  background: string
 }
 
 export class MainLayout extends Component<MainLayoutProps> {
+  getImageName(): string {
+    let imgName = 'Main'
+    switch (this.props.background) {
+      case '/':
+        imgName = 'Main'
+        break
+      case '/forum':
+        imgName = 'Main'
+        break
+      case '/login':
+        imgName = 'Login'
+        break
+      case '/leaderboard':
+        imgName = 'Leaderboard'
+        break
+      case '/profile':
+        imgName = 'Profile'
+        break
+    }
+    return imgName
+  }
+  getBackgroundImgPath() {
+    return `src/assets/images/background/${this.getImageName()}.png`
```

Comment:
И использовать здесь как
```suggestion
    return `src/assets/images/background/${imageNames[this.props.background]}.png`
```